suites/elasticTest.js

```diff
@@ -0,0 +1,21 @@
+Feature('es');
+
+const { expect } = require('chai');
+const { search } = require('../utils/elasticsearch.js');
+
+const index = 'etl_mickey_1';
+const query = '{ "query": { "nested": { "path": "BCMAMedicationLog", "query": { "range": { "BCMAMedicationLog.DosesOrdered": { "lt": 0.0029 } } } } } }';
+
+// Sample scenarios to query elasticsearch and validate results
+// scenario marked as manual to prevent it from running in CI pipeline
+Scenario('check length', async () => {
```

Comment:
Should we favour this new `.tag` chained function over the old way of tagging?

Reply:
I thought it looked nice :)

Reply:
this sample test is temporary anyway